### ISSUE TITLE
Clarify how db queries with timestamp handle short values (regarding timezone)

### ISSUE THREAD

**Issue by bernhardreiter** [Intevation/intelmq-fody-backend#27]
As request against fody-backend with a structure like

`/api/events/stats?time-observation_after=2020-02-05&time-observation_before=2020-02-06&timeres=day`

will be given to the PostgreSQL database and there interpreted as `timestamp with time zone ` value
and thus a `00:00:00` will be added and interpreted in the timezone that this database is run in.

This is in accordance with PostgreSQL's documentation, but maybe be surprising for
someone who tries to program the fody client. 

Here is a link to the documentation and how to see the timezone:
 https://www.postgresql.org/docs/9.5/runtime-config-client.html#GUC-TIMEZONE

```sql
SHOW timezone;
```

**Comment by bernhardreiter** [comment]
### Considerations
The data base schema for the `events` table comes from the intelmq postgresql backend and ideally our backend is robust against small changes in the schema.

The PostgreSQL database may also be set up earlier and for other purposes as well. It can be imagined that using a different timezone setting in the database makes sense in some use cases.
So a good solution would not pose additional requirements on the database regarding timezone or  schema.

Queries could be technically changed so that requests will always assume UTC, but this would put the burden on the client, to do a possible timezone adjustment in the interface. One solution is to let the client know in which timezone the requests will be interpreted by the database and then the client can handle it from there if necessary.

(One customer we have is close to UTC, they would be okay with only using UTC in the interface as well and they also run the database in UTC.)

**Comment by bernhardreiter** [comment]
If the timezone should be communicated to the client, the endpoint `/subqueries` could be used.

**Comment by bernhardreiter** [comment]
However it is not easy to get the full name of the timezone from the postgresql
if it is the case `localtime`. Because then it is from the underlying system, where it is also not easy.



```sql
 show timezone;
 TimeZone  
-----------
 localtime

SELECT * FROM pg_timezone_names WHERE name = current_setting('TIMEZONE');
   name    | abbrev | utc_offset | is_dst 
-----------+--------+------------+--------
 localtime | CET    | 01:00:00   | f
```

the problem is that if the system is `Europe/Berlin`, the last result line must change with the date.

And from the system it is not easy 
See considerations heres https://unix.stackexchange.com/questions/110522/timezone-setting-in-linux

also, a method with systemd could be
```sh
 timedatectl show --property=Timezone
Timezone=Europe/Berlin
```

**Comment by bernhardreiter** [comment]
Implemented with c9739ffe23799928113782283443a0587d08cf85 .

The backend will bail out, if the database does not know its full timezone name. This is sensible to do,
because newer postgresql versions (>=9.5.19) will set the timezone config parameter correctly with initdb.
So the problem will only be there with old dbs where it is fine to upgrade or with systems where guessing is
hard. In the later case we should not start guessing otherselves.

**Comment by bernhardreiter** [comment]
Also added endpoint to tickets_api (which was missing to complete this issue) with
f5da3950518ec119288ae6b03af4510da82f587d
